team/end-of-iteration-demos.md

```diff
@@ -0,0 +1,12 @@
+---
+title: End of iteration demos
+description: Time to celebrate the great work completed at the end of the iteration
+---
+
+Our team's demo session is an hour on the last day of the iteration. This is a chance to share and celebrate the amazing work done during the iteration. The purpose of this meeting is to give some time to look back at the work completed and to share it with the rest of the team. It's key that everyone is aware of what each other are working on so we don't design and develop in silos.
+
+## Demo completed work
+
+A squad should only demo completed work or work you are confident will be complete by the end of the iteration. This is because it is too tempting to share the work again in the next demo.
+
+If you are unsure refer to the [backlog sheet](https://docs.google.com/spreadsheets/d/1HH7Elz5ayh4sjH6avh0Z8_4riqlJsZ9JNMxxlesH8Go/edit) for a list of completed epics and ask other people on your squad for advice.
```

Comment:
I'm a bit worried this link will be redundant far quicker than we tend to update practice documents. Maybe don't link? Or hedge it by saying "the backlog (in [this spreadsheet] at the time of writing)" or something.

Reply:
I would hope that if we switch to another sheet we would add a note and link at the top for clarity.